CRM/Utils/Date.php

```diff
@@ -1258,6 +1258,16 @@ public static function relativeToAbsolute($relativeTerm, $unit) {
                   $from['Y'] -= ($relativeTermSuffix - 1);
                 }
                 break;
+
+              case 'earlier':
+                $to['d'] = $now['mday'];
+                $to['M'] = $now['mon'];
```

Comment:
I don't think this is correct. Compare to line 1157. If I do earlier_2.year it says Dec 9, 2019, but it should be Dec 31, 2019.